tests/markdown/README.md

```diff
@@ -388,7 +388,7 @@ and add this config to your `package.json`:
   "lint-staged": {
     "*.{js,json,css}": [
       "prettier --write",
-      "git add"
+      "git add" // Warning: This breaks staging specific lines. See footnote below.
```

Comment:
You've edited the input to a test without changing its expected output, which is why Travis fails. I'm not sure we should update the test, though.